database/migrations/2014_10_12_100000_create_password_resets_table.php

```diff
@@ -15,7 +16,7 @@ public function up()
         Schema::create('password_resets', function (Blueprint $table) {
             $table->string('email')->index();
             $table->string('token')->index();
-            $table->timestamp('created_at');
+            $table->timestamp('created_at')->nullable();
```

Comment:
Is modifying an already-run migration going to cause issues? Can't remember this one